scripts/validate_data.py

```diff
@@ -54,7 +55,22 @@ def validate_data(data_directory):
     # Calculate actual hash for given filename.
     # If hash for filename is not the same as the one in the file, raise
     # ValueError
-    raise NotImplementedError('This is just a template -- you are expected to code this.')
+    hash_file_path = os.path.join(data_directory, "group-00/hash_list.txt")
+    with open(hash_file_path) as f:
+        lines = f.readlines()
```

Comment:
```suggestion
        lines = f.read().splitlines()
```
Reading the text file like this will automatically remove the '\n' at the end of each line.

Reply:
this would mean also removing the `replace("\n", "") at line 66 @Cionkito, can you edit your suggestion to incorporate both?

Reply:
Absolutely, my bad.